pytorch_lightning/accelerators/gpu.py

```diff
@@ -130,5 +134,5 @@ def _get_gpu_id(device_id: int) -> str:
     """Get the unmasked real GPU IDs."""
     # All devices if `CUDA_VISIBLE_DEVICES` unset
     default = ",".join(str(i) for i in range(torch.cuda.device_count()))
-    cuda_visible_devices: List[str] = os.getenv("CUDA_VISIBLE_DEVICES", default=default).split(",")
+    cuda_visible_devices = os.getenv("CUDA_VISIBLE_DEVICES", default=default).split(",")
```

Comment:
Unrelated to this PR, but seems we could consolidate/unify the device related functions a bit. 
We have similar logics in https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/device_parser.py too

Reply:
Want to open an issue with the precise proposal?

Reply:
Issue created https://github.com/PyTorchLightning/pytorch-lightning/issues/11427